play/templates/base.html

```diff
@@ -44,23 +45,6 @@
   {% block content %}{% endblock %}
 </body>
 <script>
-  (function() {
```

Comment:
I decided to remove this, it was causing some oddities with the server-side form validation